src/lib/style/components/entity_finder/_entity-finder.scss

```diff
@@ -7,16 +7,12 @@ $-header-height: 45px;
 $-border: $-border-size solid SRC.$border-color-gray;
 $-finder-height: 500px;
 $-breadcrumb-height: 40px;
-$-selected-color: white;
-$-selected-background-color: SRC.$primary-action-color-active;
+$-selected-background-color: map.get(SRC.$secondary-color-palette, 200);
 $-selected-hover-background-color: color.adjust(
   $-selected-background-color,
-  $lightness: -5%
-);
-$-hover-background-color: color.change(
-  SRC.$primary-action-color-active,
-  $alpha: 0.2
+  $lightness: -10%
 );
+$-hover-background-color: map.get(SRC.$primary-color-palette, 100);
```

Comment:
Style updates for the Entity Finder to match designs, and not conflict with the new checkbox

https://user-images.githubusercontent.com/17580037/162285895-0d433837-ad9e-46a1-a43f-9137cc6f46a7.mov